DirectEmployers/MyJobs
# [PD 3123] match table layout cbo

## What's this PR do?
Matches the table layout inside of CBO with the tables inside of User Management

## Screenshots (if appropriate)
![screen shot 2017-04-07 at 4 47 52 pm](https://cloud.githubusercontent.com/assets/22029077/24819343/34a3897e-1bb2-11e7-8e6a-f78480fbb23d.png)
![screen shot 2017-04-07 at 4 48 03 pm](https://cloud.githubusercontent.com/assets/22029077/24819342/349f76f4-1bb2-11e7-8f27-ade7ccc7cee1.png)
![screen shot 2017-04-07 at 4 48 10 pm](https://cloud.githubusercontent.com/assets/22029077/24819345/38173fc4-1bb2-11e7-8e67-30559b11bdaa.png)
![screen shot 2017-04-07 at 4 48 21 pm](https://cloud.githubusercontent.com/assets/22029077/24819346/3817f9b4-1bb2-11e7-9ea9-bd044bffd28d.png)


## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
Pull down the repository. You will go into the CBO app and on the main page see the main table. Along with that if you edit a CBO there will be a table of contacts on the next page. If you go even further to edit a contact the last tables will be on the edit contact page with Distribution Location and Rules

## Any background context you want to provide?
